diff --git a/node_modules/moti/build/skeleton/skeleton-new.js b/node_modules/moti/build/skeleton/skeleton-new.js
index 9692965..b543772 100644
--- a/node_modules/moti/build/skeleton/skeleton-new.js
+++ b/node_modules/moti/build/skeleton/skeleton-new.js
@@ -2,7 +2,7 @@ import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import React, { createContext, useContext } from 'react';
 import { View, StyleSheet } from 'react-native';
 import { useAnimatedStyle, useDerivedValue, useSharedValue, } from 'react-native-reanimated';
-import { View as MotiView } from '../components';
+import { View as MotiView } from '../components/view';
 import { DEFAULT_SKELETON_SIZE as DEFAULT_SIZE, defaultDarkColors, defaultLightColors, baseColors, } from './shared';
 export default function Skeleton(props) {
     const skeletonGroupContext = useContext(SkeletonGroupContext);
diff --git a/node_modules/moti/build/skeleton/skeleton.js b/node_modules/moti/build/skeleton/skeleton.js
index 4e69103..f0fa7ee 100644
--- a/node_modules/moti/build/skeleton/skeleton.js
+++ b/node_modules/moti/build/skeleton/skeleton.js
@@ -2,7 +2,7 @@ import { jsx as _jsx, jsxs as _jsxs } from "react/jsx-runtime";
 import { LinearGradient } from 'expo-linear-gradient';
 import React, { useState, createContext, useContext } from 'react';
 import { View, StyleSheet } from 'react-native';
-import { View as MotiView } from '../components';
+import { View as MotiView } from '../components/view';
 import { AnimatePresence } from '../core';
 import { DEFAULT_SKELETON_SIZE as DEFAULT_SIZE, defaultDarkColors, defaultLightColors, baseColors, } from './shared';
 export default function Skeleton(props) {
diff --git a/node_modules/moti/src/skeleton/skeleton-new.tsx b/node_modules/moti/src/skeleton/skeleton-new.tsx
index 32ac3da..af30648 100644
--- a/node_modules/moti/src/skeleton/skeleton-new.tsx
+++ b/node_modules/moti/src/skeleton/skeleton-new.tsx
@@ -7,7 +7,7 @@ import {
 } from 'react-native-reanimated'
 import type Animated from 'react-native-reanimated'
 
-import { View as MotiView } from '../components'
+import { View as MotiView } from '../components/view'
 import { MotiTransitionProp } from '../core'
 import {
   DEFAULT_SKELETON_SIZE as DEFAULT_SIZE,
diff --git a/node_modules/moti/src/skeleton/skeleton.tsx b/node_modules/moti/src/skeleton/skeleton.tsx
index 3ef5dd1..8bcd147 100644
--- a/node_modules/moti/src/skeleton/skeleton.tsx
+++ b/node_modules/moti/src/skeleton/skeleton.tsx
@@ -2,7 +2,7 @@ import { LinearGradient } from 'expo-linear-gradient'
 import React, { useState, createContext, useContext } from 'react'
 import { View, StyleSheet } from 'react-native'
 
-import { View as MotiView } from '../components'
+import { View as MotiView } from '../components/view'
 import { AnimatePresence, MotiTransitionProp } from '../core'
 import {
   DEFAULT_SKELETON_SIZE as DEFAULT_SIZE,
